app/controllers/votes_controller.rb

```diff
@@ -1,8 +1,15 @@
 class VotesController < ApplicationController
-  def create
```

Comment:
You should keep `def create` here, it's more RESTful to have a `create` action on the `VotesController`.

http://guides.rubyonrails.org/routing.html